core/domain/user_services_test.py

```diff
@@ -132,6 +132,40 @@ def test_get_user_id_from_username(self):
         self.assertIsNone(
             user_services.get_user_id_from_username('fakeUsername'))
 
+    def test_set_and_get_user_email_preferences(self):
+        user_id = 'someUser'
```

Comment:
I meant 'someUser' --> 'userId'.

Having said that, I realize there are other tests in this file which already use 'someUser', so I retract my comment. Thanks for pushing back!